Dart/testSrc/com/jetbrains/lang/dart/typing/DartTypingTest.java

```diff
@@ -607,4 +607,11 @@ public void testEnterAfterEqualsProperty() {
                  "o.x =\n" +
                  "    <caret>");
   }
+
+  public void testEnterBeforeStringParam() {
```

Comment:
@stevemessick ok, as this is an important fix, I fixed the test myself and pushed. Will cherry-pick to the release branch.